src/components/Card/Card.js

```diff
@@ -0,0 +1,39 @@
+import PropTypes from 'prop-types';
+import styled, { css } from 'react-emotion';
+
+import { childrenPropType } from '../../util/shared-prop-types';
+import { shadowSingle } from '../../styles/style-helpers';
+
+const baseStyles = ({ theme }) => css`
+  label: card;
+  background-color: ${theme.colors.white};
+  border-radius: ${theme.borderRadius.mega};
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  ${shadowSingle({ theme })};
+  min-width: 460px;
+  padding: ${theme.spacings.mega} ${theme.spacings.giga};
+`;
+
+/**
+ * Card component that is used for displaying content on a grid.
+ */
+const Card = styled('div', { label: 'Card' })(baseStyles);
```

Comment:
Can you explain why here you have `{ label: 'Card' }` but the baseStyles also specifies a label of card (lowercase)?

Reply:
There's an explanation of the current label situation [in the docs](https://sumupteam.atlassian.net/wiki/spaces/DEV/pages/223871058/Circuit+UI+Developer+guide) under "Using labels" (I don't know how to link to anchors in Confluence 😞).

Short version:
  - Put labels for BEM classes into the style functions.
  - Put `label` property into the styled options to have the value as component `displayName`. Otherwise displayName will be `styled({element name})`. We will be able to automate this by adding and configuring the emotion babel plugin.